### PR TITLE
Docs: update commitment to OSS

### PR DESCRIPTION
**Pull request by ozkatz**

### CODE REVIEW

File: docs/commitment.md [hunk 1]
```diff
@@ -16,31 +16,30 @@ The project was created and is supported by Treeverse, a commercial company, fou
...
 
-Well, it will take a while to figure out, and currently everything we do is open source, but in the future, we will provide a commercial offering according to users’ feedback.
-For example, SLA based support was already requested by some of our design partners. Apparently some organizations feel comfortable with being early adopters of technology if they have the guarantee that when an issue arises someone is committed contractually to resolve it within a given SLA. If this is your preference, we are here for you.
-In addition, in the future, we may choose to provide usability features over the core capabilities, enterprise features revolving around user management and security, and a SaaS offering.  
+lakeFS Cloud's vision is to provide a managed, usable data versioning and management solution for data practitioners, by leveraging the open source technology behind lakeFS and building added capabilities and unique features on top.
 
+As part of our commitment to the open source values of transparency and interoperability, we believe everyone should be able to enjoy these benefits, regarless of whether or not they choose to use the managed offering.
```
Comment (peacing) [inline docs/commitment.md:43]: small typo: regarless >> regardless

File: docs/commitment.md [hunk 2]
```diff
@@ -16,31 +16,30 @@ The project was created and is supported by Treeverse, a commercial company, fou
...
-![Open Core Model]({{ site.baseurl }}/assets/img/open_core.png)
+We believe that every engineer should be able to use, contribute and influence cutting edge technologies, so they can innovate in their domain.
 
 ***What is our commitment to open source?***
 
-We define lakeFS, our open source project, as the data lake management platform that provides Git-like operations over an object-storage, that can be utilized by any applications accessing the data lake. 
-In other words, the following will remain part of lakeFS:
+We created lakeFS, our open source project, to provide a Git-like interface on top of object stores - that you can fully take advantage of with any data application at any scale.
+
+In other words, we commit that the following capabilities are *and will remain* open source as part of lakeFS:
```
Comment (johnnyaug) [inline docs/commitment.md:25]:
```suggestion
For that reason, we commit that the following capabilities are *and will remain* open source as part of lakeFS:
```

File: docs/commitment.md [hunk 3]
```diff
@@ -16,31 +16,30 @@ The project was created and is supported by Treeverse, a commercial company, fou
...
+
+***What is lakeFS Cloud?***
 
-***What will be included in the commercial layer?***
+Treeverse offers lakeFS Cloud, which provides all the same benefits of the Git-like interface on top of object stores as a fully-managed service.
 
-Well, it will take a while to figure out, and currently everything we do is open source, but in the future, we will provide a commercial offering according to users’ feedback.
-For example, SLA based support was already requested by some of our design partners. Apparently some organizations feel comfortable with being early adopters of technology if they have the guarantee that when an issue arises someone is committed contractually to resolve it within a given SLA. If this is your preference, we are here for you.
-In addition, in the future, we may choose to provide usability features over the core capabilities, enterprise features revolving around user management and security, and a SaaS offering.  
+lakeFS Cloud's vision is to provide a managed, usable data versioning and management solution for data practitioners, by leveraging the open source technology behind lakeFS and building added capabilities and unique features on top.
```
Comment (johnnyaug) [inline docs/commitment.md:41]:
```suggestion
The vision of lakeFS Cloud is to provide a managed data versioning and management solution for data practitioners. lakeFS Cloud  will leverage the lakeFS open source technology and add capabilities and unique features on top of it.
```

**Review by johnnyaug — CHANGES_REQUESTED** [review]

**Review by johnnyaug — APPROVED** [review]

**Review by nopcoder — APPROVED** [review]
